src/js/generalFunctions.js

```diff
@@ -0,0 +1,7 @@
+function generateDomElement(currentNode, element, ...classes) {
```

Comment:
Could you rename this file ?
e.g:
Utils.js
Helpers.js

Reply:
Done